runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -90,6 +90,26 @@
     @Getter
     private boolean logHasNoCheckPoint = false;
 
+    private boolean whiteList = false;
```

Comment:
I used it as the knob for the mutual exclusion between the whitelist mode and the blacklist mode. If the client of the API wants to do whitelist but because of its logic he doesn't have any table in the provided streamsToLoad, it should still be a whitelist. 

As is, I could not infer from the streasToLoad if we are in whitelist mode with no streams given or if we are in blacklist mode.

Reply:
whiteList is set to true when you add a stream to a white list, so it can be inferred from the list size. Setting it to true even if the white list is empty is another issue.

Also, my initial point was its kind of redundant and it doesn't seem to work. Consider this example:

@Test
    public void whiteListMode() throws Exception {
        populateMaps(MORE, getDefaultRuntime(), CorfuTable.class, true, SOME);

        List<String> streamsToLoad = new ArrayList<>();
        streamsToLoad.add("Map1");
        streamsToLoad.add("Map3");

        CorfuRuntime rt2 = new CorfuRuntime(getDefaultConfigurationString()).connect();
        FastObjectLoader loader = new FastObjectLoader(rt2)
                .setDefaultObjectsType(CorfuTable.class);
        loader.addStreamToIgnore("a");
        loader.addStreamsToLoad(streamsToLoad);
        loader.loadMaps();

        assertThatMapsAreBuiltWhiteList(rt2, streamsToLoad);
    }